controller/deployments.go

```diff
@@ -355,6 +361,69 @@ func (c *DeploymentsController) ShowSpace(ctx *app.ShowSpaceDeploymentsContext)
 	// Kubernetes doesn't know about space ID, so add it here
 	space.ID = ctx.SpaceID
 
+	if checkPerms {
+		// next, try to get permissions
+		guid := goauuid.UUID(ctx.SpaceID)
+
+		// permission for all deployments
+		for _, appl := range space.Attributes.Applications {
+			for _, depl := range appl.Attributes.Deployments {
+				deployPath := witclient.SetDeploymentDeploymentsPath(guid, appl.Attributes.Name, depl.Attributes.Name)
+				deployURLStr := rest.AbsoluteURL(ctx.Request, deployPath)
+				if err != nil {
+					return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "could not retrieve path for %s", *kubeSpaceName))
+				}
+				envName := depl.Attributes.Name
+				methods := []string{}
+				canScale, err := kc.CanScaleDeployment(envName)
+				if err != nil {
+					return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "could not retrieve permission for %s", *kubeSpaceName))
+				}
+				if canScale {
+					methods = append(methods, "PATCH")
+				}
+				canDelete, err := kc.CanDeleteDeployment(envName)
+				if err != nil {
+					return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "could not retrieve permission for %s", *kubeSpaceName))
+				}
+				if canDelete {
+					methods = append(methods, "DELETE")
+				}
+				// NOTE: we don't actually allow GET to this endpoint, so don't claim we do by adding "GET" to methods
+				depl.Links.Self = &app.LinkWithAccess{
+					Href: &deployURLStr,
+					Meta: &app.EndpointAccess{
+						Methods: methods,
+					},
+				}
```

Comment:
We could also include links to the `stats` and `statseries` endpoints here.

Reply:
Good idea - done.